### ISSUE TITLE
When Updating AdGuard For Android Adblocker, the popup open doesn't work. 

### ISSUE THREAD

**Issue by Batman2814** [AdguardTeam/AdguardForAndroid#5021]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

AdGuard 4.3 Nightly 7 for Android Adblocker 

### Environment

```markdown
- OS: Android 13
- Device: Samsung Galaxy S22 Ultra 5G
- Firmware:
```

### Root access

- [ ] Yes, I have it.

### What filters do you have enabled?

AdGuard Base filter, AdGuard Mobile Ads filter, AdGuard Chinese filter, AdGuard Tracking Protection filter, AdGuard URL Tracking filter, AdGuard Social Media filter, AdGuard Cookie Notices filter, AdGuard Popups filter, AdGuard Mobile App Banners filter, AdGuard Other Annoyances filter, AdGuard DNS filter, EasyPrivacy, Fanboy's Annoyances, Fanboy's Social Blocking List, Peter Lowe's Blocklist, Online Malicious URL Blocklist, Fanboy's Enhanced Tracking List, I don't care about cookies, EasyList Cookie List, NoCoin Filter List, Legitimate URL Shortener

### What Stealth Mode options do you have enabled?

Hide your search queries, Send Do-Not-Track signals, Strip URLs of tracking parameters, Self-destruction of third-party cookies, Disable WebRTC, Block Push API, Block Location API, Hide your Referrer from third-parties, Hide your User Agent, Hide your IP address, Remove X-client-Data header from HTTP request, Protect from DPI

### Issue Details

Steps to reproduce:
1. UPDATE AdGuard for Android Adblocker when available 
2. Wait for pop to show once update it done. 
3. Click open 

### Expected Behavior

Should open AdGuard for Android Adblocker when clicked once update is done. 

### Actual Behavior

When click open the App AdGuard for Android Adblocker doesn't open. 

### Screenshots

![Screenshot_20231010_114947_Package installer](https://github.com/AdguardTeam/AdguardForAndroid/assets/96619913/44e15cf8-36b4-4405-b197-33fd63dfcfc6)

### Additional Information

you guys told me that before this was an issue on the software firmware from my phone brand, samsung. I can tell you for sure that it's not an issue with Samsung Update firmware that it's currently running because when I was on the previous version of Adguard beginning of the year the firmware stayed the same from Samsung side and there where a few updates that were released on Adgaurd for Andoid side before the next firmware for the phone changed.  So, on those versions of Adguard, it was working when I clicked open once the update was done, but somehow, the bug in Adgaurd keeps causing problems when you keep updating Adguard for Android. I really hope that you guys could fix this issue because this it not the first case I have opened about this problem.

**Comment by maxikuzmin** [comment]
@Batman2814 duplicate of #4624 

This is a native Android issue, reproduces on API 30 and above. We have reported this bug to the android developers and are currently waiting for a fix.

https://issuetracker.google.com/issues/280783677
